modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,157 @@
+* You must know the names or IDs of the subnets to which you intend to map your IngressController.
+
+.Procedure
+To create an Ingress Controller with manually specified load balancer subnets, you can follow these steps:
+
+. Create a custom resource (CR) file.
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name>
+spec:
+  domain: <domain>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+  dnsManagementPolicy: Managed
+----
+
+. Create a custom resource (CR) file.
+
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name> <1>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic
+          classicLoadBalancer: <3>
+            subnets:
+              ids: <4>
+              - <subnet> <5>
+              - <subnet>
+              - <subnet>
+----
+<1> Replace `<name>` with a name for the Ingress Controller
+<2> Replace `<domain>` with the DNS name serviced by the Ingress Controller
+<3> You can also use the `networkLoadBalancer` field if using an NLB
+<4> You can optionally specify subnet by name using the `names` field
+<5> Specify subnet IDs (or names if you using `names`).
++
+[IMPORTANT]
+====
+You can specify a maximum of one subnet per availability zone. Only provide public subnets for external Ingress Controllers and private subnets for internal Ingress Controllers.
+====
++
+.. Apply the CR file:
+
+. Save the file and apply it using the OpenShift CLI:
++
+[source,terminal]
+----
+$  oc apply -f sample-ingress.yaml
+----
+.. Confirm the load balancer was provisioned successfully by checking the IngressController conditions. 
++
+[source,terminal]
+----
+$ oc get ingresscontroller -n openshift-ingress-operator <name> -o jsonpath="{.status.conditions}" | yq -PC
+----
++
+= Updating the subnets on an existing Ingress Controller
+
+. Choose the subnets.
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic <1>
+          classicLoadBalancer:
+            subnets:
+              ids:
+              - <updated_subnet>
+              - <updated_subnet>
+              - <updated_subnet>
+----
+<1> Specify updated subnet IDs (or names if you using `names`).
```

Comment:
I think it would be helpful if you replicate the same callouts from the previous example here, just in the off chance someone comes back to reference this but doesn't look at the other one before, they may have the question of whether they can use names instead of IDs, and that was covered in the other callouts.